packages/oraidex-common/src/network.ts

```diff
@@ -705,6 +869,31 @@ export const chainInfos: CustomChainInfo[] = [
       txUrl: "https://www.mintscan.io/noble/txs/{txHash}"
     }
   },
+  {
+    rpc: "https://rpc-cosmos.oraidex.io",
+    rest: "https://lcd-cosmos.oraidex.io",
+    chainId: "cosmoshub-4",
+    chainName: "Cosmos Hub",
+    networkType: "cosmos",
+    bip44: {
+      coinType: 118
+    },
+    Icon: AtomIcon,
+    IconLight: AtomIcon,
+    bech32Config: defaultBech32Config("cosmos"),
+    feeCurrencies: [AtomToken],
+    currencies: [
+      {
+        coinDenom: "ATOM",
```

Comment:
Also the ATOM token as well. I remember that we have already defined tokens to use in these cases like OraiToken, AtomToken,...